docs/data/getting-started/support/support.md

```diff
@@ -0,0 +1,35 @@
+---
+title: MUI X - Support
+---
+
+# MUI X - Support
+
+<p class="description">Learn how to get support for MUI X components, including feature requests, bug fixes, answers to how-to questions, and professional support from our team.</p>
+
+## GitHub
+
+We use GitHub issues as a bug and feature request tracker. If you think you have found a bug, or have a new feature idea, please start by making sure it hasn't already been [reported or fixed](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed). You can search through existing issues and pull requests to see if someone has reported one similar to yours.
+
+[Open an issue](https://github.com/mui/mui-x/issues/new/choose) in the MUI X repo.
+
+## Stack Overflow
+
+Visit Stack Overflow to ask questions and read crowdsourced answers from expert developers in the MUI community, as well as MUI maintainers.
+
+[Post a question about MUI X](https://stackoverflow.com/questions/tagged/mui) on Stack Overflow.
+
+## Professional support
+
+When purchasing an MUI X Pro or Premium license you get access to professional support for a limited duration.
+Support is available on multiple channels, but the recommended channel is GitHub issues.
+You can also reach us by email at x@mui.com if you need to share private information.
+
+- **MUI X Pro**: No SLA is provided but MUI's maintainers give these issues more attention than the ones from the Community plan. The channels:
+  - GitHub: [Open a new issue](https://github.com/mui/mui-x/issues/new/choose) and leave your Order ID.
+  - Email (only to share private information): [Open a new issue](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email at x@mui.com.
```

Comment:
```suggestion
```
cc. @oliviertassinari

Reply:
Commented in https://github.com/mui/material-ui/pull/32811#discussion_r884191200